src/Microsoft.Health.Dicom.Functions.Client.UnitTests/DicomAzureFunctionsHttpClientTests.cs

```diff
@@ -8,23 +8,26 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
+using System.Net.Http.Json;
 using System.Net.Mime;
-using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Options;
 using Microsoft.Health.Dicom.Core.Exceptions;
 using Microsoft.Health.Dicom.Core.Messages.Operations;
 using Microsoft.Health.Dicom.Core.Models.Operations;
 using Microsoft.Health.Dicom.Functions.Client.Configs;
-using Newtonsoft.Json;
 using Xunit;
 
 namespace Microsoft.Health.Dicom.Functions.Client.UnitTests
 {
     public class DicomAzureFunctionsHttpClientTests
     {
-        private static readonly IOptions<FunctionsClientOptions> DefaultConfig = Options.Create(
+        private readonly IOptions<JsonSerializerOptions> _jsonSerializerOptions;
+
+        private static readonly IOptions<FunctionsClientOptions> DefaultOptions = Options.Create(
```

Comment:
going forward are we changing all Config to be Options?

Reply:
I think generally we should create a distinction between the concept of "options," a collection of settings, and a "config," a collection of settings sourced from a particular storage medium such as a file. In .NET Core, it appears these settings are often called "options" and are loaded into the DI container via the `IOptions<T>`, `IOptionsSnapshot<T>`, and `IOptionsMonitor<T>`. It also just so happens that the data in an `IOptions<T>` instance is sourced from an `IConfiguration` object, but it is not a requirement.

Reply:
should we rename this then?
![image](https://user-images.githubusercontent.com/26657972/127395042-975bafe3-69cd-4ef8-b562-8017ce6866cd.png)

Reply:
oh oops nevermind

Reply:
Yeah, I think that makes sense